module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,24 @@
+#include <algorithm>
+#include <forward_list>
+#include <string>
+
+#include "sort.hpp"
+
+std::deque<std::string> lengthSort(std::forward_list<std::string>& list){
+    list.sort();
+    std::deque<std::string> deq{};
+    for(auto& listElement : list){
+        deq.push_back(listElement);
+    }
```

Comment:
You can initialize deque  using iterator `std::deque<std::string> deq{list.begin(), list.end()};`